tests/conftest.py

```diff
@@ -0,0 +1,18 @@
+import pytest
+from selenium import webdriver
+from selenium.webdriver.common.by import By
+
+@pytest.fixture
+def login(base_url, selenium):
+    selenium.get('{0}/admin/login/'.format(base_url))
+    selenium.maximize_window()
+
+    # login as admin
+    username = selenium.find_element_by_id('id_username')
+    password = selenium.find_element_by_id('id_password')
+    login = selenium.find_element_by_class_name('submit-row')
+    username.send_keys("admin")
+    password.send_keys("admin")
+    login.click()
+
+    return login
```

Comment:
You're returning a reference to the login button HTML element. That element is on the login page, which you leave by clicking the button two lines above. The reference thus can't really be useful to tests depending on this fixture, can it?

Not all fixtures need to return a value. Just performing some tasks to change global state is fine, too.